src/index.js

```diff
@@ -0,0 +1,150 @@
+        sky.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧'
+    } else if (skyValue === 'snowy') {
+        sky.textContent = '❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️'
+    }
+};
+
+
+const updatelandScape = () => {
+    const sky = document.getElementById('sky');
+    if (state.temp > 80) {
+        tempInfo.style.color = 'red'
+        sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️'
+        landScape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂'
+    } else if (state.temp > 70) {
+        tempInfo.style.color = 'orange'
+        sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️'
+        landScape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷'
+    } else if (state.temp > 60) {
+        tempInfo.style.color = 'yellow'
+        sky.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️'
+        landScape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃'
+    } else if (state.temp > 50) {
+        tempInfo.style.color = 'green'
+        sky.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧'
+        landScape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲'
+    } else if(state.temp < 49) {
+        tempInfo.style.color = 'teal'
+        landScape.textContent = '❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️'
+    }
+};
+
+const findLatitudeAndLongitude = () => {
+    let latitude, longitude;
+    axios
+    .get('https://weather-report-proxy-server.herokuapp.com/location', {
+        params: {
+        q: state.city,
+        },
+    })
+    .then((response) => {
+        latitude = response.data[0].lat;
+        longitude = response.data[0].lon;
+
+        // make the next API call here!
+        findTemperature(latitude, longitude);
+    })
+    .catch((error) => {
+        console.log('error in findLatitudeAndLongitude!');
+    });
+};
+
+const findTemperature = (latitude, longitude) => {
+    axios
+    .get('https://weather-report-proxy-server.herokuapp.com/weather', {
+        params: {
+        lat: latitude,
+        lon: longitude,
+        },
+    })
+    .then((response) => {
+        const kelvin = response.data['current']['temp'];
+        state.temp = convertKelvinToFahrenheit(kelvin);
+        updateWeather();
+    })
+    .catch((error) => {
+        console.log('error in weather info!');
+    });
+};
+
+const convertKelvinToFahrenheit = (kelvin) => {
+    let fahrenheit;
+    fahrenheit = parseInt(1.8 * (kelvin - 273) + 32);
+    return fahrenheit;
+};
+
+
+const registerEventHandlers = () => {
+    updatelandScape();
+    tempUp.addEventListener('click', increaseWeather);
+    tempDown.addEventListener('click', decreaseWeather);
+    cityInput.addEventListener('input', updatecityInput);
+    realTemp.addEventListener('click', findLatitudeAndLongitude);
+    skySelect.addEventListener('change', updateSky);
+    resetCity.addEventListener('click', updateresetCity);
+};
+
+document.addEventListener('DOMContentLoaded', registerEventHandlers);
+
+// Generally we would register event handlers after the event
+// DOMContentLoaded but this does not work on codesandbox
+// document.addEventListener("DOMContentLoaded", registerEventHandlers);
+
+// registerEventHandlers();
+// console.log("loaded");
```

Comment:
Remove comments, unused code that's been commented out, and debugging print statements to keep your code concise and readable.